src/hotspot/share/gc/g1/g1RemSetTrackingPolicy.cpp

```diff
@@ -66,14 +66,12 @@ static void print_before_rebuild(HeapRegion* r, bool selected_for_rebuild, size_
                                   "total_live_bytes %zu "
                                   "selected %s "
                                   "(live_bytes %zu "
-                                  "marked %zu "
                                   "type %s)",
                                   r->hrm_index(),
                                   p2i(r->top_at_mark_start()),
                                   total_live_bytes,
                                   BOOL_TO_STR(selected_for_rebuild),
                                   live_bytes,
-                                  r->marked_bytes(),
```

Comment:
At this point (scrubbing start) `marked_bytes()` contain the amount of marked bytes in the _previous_ marking. This is not particularly interesting to see; `live_bytes` contains the current marked bytes below tams and is printed.